### ISSUE TITLE
US22 - As a user, I would like to connect my direction to next class to my google calendar.

### ISSUE THREAD

**Issue by bkenza** [mohanadarafe/GuideMe#31]
The user should be able to connect (integrate) the direction to its next class to its google calendar for more accessibility.
